README.md

```diff
@@ -195,13 +195,23 @@ wav         - speech files in wave file format
 
 ## Building for Windows on a Linux machine
 
-On Ubuntu 17 and above:
+On Ubuntu 20.04 LTS ( for 32bit Windows):
    ```
-   sudo apt-get install mingw-w64
+   sudo apt install mingw-w64 g++-mingw-w64
    mkdir build_windows && cd build_windows
-   cmake .. -DCMAKE_TOOLCHAIN_FILE=/home/david/freedv-dev/cmake/Toolchain-Ubuntu-mingw32.cmake -DUNITTEST=FALSE -DGENERATE_CODEBOOK=/home/david/codec2/build_linux/src/generate_codebook 
+   cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-Ubuntu-mingw32.cmake .. 
```

Comment:
@DJ2LS pls leave the `-DGENERATE_CODEBOOK` codebook logic in this PR as I have https://github.com/drowe67/codec2/pull/285 in progress that removes the static versions of the codebook files from the source.  So this logic will soon be needed, even though it is optional at present.